packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -64,8 +64,6 @@ export interface IFluidDataStoreRuntime extends
 
     readonly existing: boolean;
 
-    readonly parentBranch: string | null;
```

Comment:
I'd add this to breaking.md. Chances that we will break someone are extremely low, but who knows.

Reply:
Added in latest, will wait for the 0.30 version bump before merging.